src/game.cpp

```diff
@@ -3552,6 +3552,16 @@ void Game::processClientEvents(CameraOrientation *cam, float *damage_flash)
 			bool enable = event.override_day_night_ratio.do_override;
 			u32 value = event.override_day_night_ratio.ratio_f * 1000;
 			client->getEnv().setDayNightRatioOverride(enable, value);
+		} else if (event.type == CE_SET_CLOUDS) {
```

Comment:
Are you talking about the complete event type dispatching here? That could in fact be done if the very first condition is split up (it combines a type check and a getHP check), but seems very out of scope for this pull request. A separate one, perhaps?